backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -748,6 +848,13 @@ public RequestStatus updateProjectVulnerabilityRating(ProjectVulnerabilityRating
         return projects;
     }
 
+    private List<Release> getDirectlyLinkedReleasesInNewState(Project project) {
+        Set<String> releaseIds = CommonUtils.nullToEmptyMap(project.getReleaseIdToUsage()).keySet();
+        List<Release> releases = componentDatabaseHandler.getReleasesForClearingStateSummary(releaseIds);
+        return releases.stream().filter(release -> null == release.getClearingState()
+                || ClearingState.NEW_CLEARING.equals(release.getClearingState())).collect(Collectors.toList());
```

Comment:
There is a possibility that **Release** `clearingState` is `null` in db, and in such scenario **Release** will be considered with `clearingState` as `NEW`.

This will not meet the required condition.
> release.getClearingState()== ClearingState.NEW_CLEARING